docs/buildspecs/performance_checks.rst

```diff
@@ -88,6 +88,41 @@ Let's run ``buildtest inspect query -o stream_test`` to retrieve the test detail
 
     .. command-output:: buildtest inspect query -o stream_test
 
+Assert Less Than Equal
+----------------------
+
+buildtest can determine status check based on performance check. In this next example, we will run the
+`STREAM <https://www.cs.virginia.edu/stream/>`_ memory benchmark and capture :ref:`metrics <metrics>` named ``copy``, ``scale``
+``add`` and ``triad`` from the output and perform an Assertion Less Than Equal (``assert_le``) with a reference value.
+
+The ``assert_le`` contains a list of assertions where each metric name is
+referenced via ``name`` that is compared with the reference value defined by ``ref`` property. The comparison
+is ``metric_value <= ref``, where **metric_value** is the value assigned to the metric name captured by the regular
+expression. The ``type`` field in the metric section is used for the type conversion which can be **float**, **int**, or **string**.
+The ``item`` is a numeric field used in `match.group <https://docs.python.org/3/library/re.html#re.Match.group>`_ to retrieve the output
+from the regular expression search. The item must be non-negative number.
+
+.. literalinclude:: ../tutorials/perf_checks/assert_le.yml
+    :language: yaml
+    :emphasize-lines: 12-46
+    :linenos:
+
+
+buildtest will evaluate each assertion in the list and use a logical AND to determine the final
+status of ``assert_le``.
+
+Let's build this test, take a close look at the output of ``buildtest build`` and take note of the assertion
+statement.
+
+.. dropdown:: ``buildtest build -b tutorials/perf_checks/assert_le.yml``
+
+    .. command-output:: buildtest build -b tutorials/perf_checks/assert_le.yml
+
+
+.. dropdown:: ``buildtest inspect query -o assert_le_example``
+
+    .. command-output:: buildtest inspect query -o assert_le_example 
+
```

Comment:
i think this text was a copy of the another section in docs in that case i am leaning on having one example that shows all the keys `assert_le`, `assert_lt`, `assert_gt` and `assert_ge` so we dont have to rewrite docs for each section.

Reply:
If that's the case, we probably need to change ``Assert Greater Equal`` title to something like Performance check and then put 4 tests below.

Reply:
yeah i agree with you, let's try to update that part of docs as a separate PR until we get all the implementation added for all the relevant checks that way we can just document once for the comparison checks `>`, `>=`, `<=`, `<`.